### DISCUSSION TITLE
Copy version number to clipboard when clicking on it

### DISCUSSION THREAD

**Discussion by minetoblend** [ppy/osu#14371] [Ideas] [answered]
Having to type out the version number each time when filing an issue is kind of bothersome. Being able to copy it to the clipboard by clicking on the version number at the bottom of the main menu would be a nice addition.

**Comment by peppy** [comment, marked as answer]
You can click it at the bottom of settings for now. It will take you to the changelog page with the version number.

Eventually we will add a better issue reporting system.
